frontend/packages/process-editor/src/components/ConfigPanel/ConfigContent/EditDataType/SelectDataType.module.css

```diff
@@ -0,0 +1,11 @@
+.buttons {
+  display: flex;
+  gap: var(--fds-spacing-3);
+  padding-top: var(--fds-spacing-3);
+}
+
+.dataTypeSelect {
+  padding-left: var(--fds-spacing-3);
+  padding-right: var(--fds-spacing-3);
+  gap: 4px;
```

Comment:
should this gap also have spacing token? 😄

Reply:
Not needed at all actually, so removed 😎